backend/api/services.py

```
@@ -0,0 +1,133 @@
+    
+    @staticmethod
+    def get_storage_credentials():
+        return {
+            'account_name': os.environ.get('AZURE_STORAGE_ACCOUNT_NAME'),
+            'account_key': os.environ.get('AZURE_STORAGE_ACCOUNT_KEY'),
+            'video_container': os.environ.get('AZURE_VIDEO_CONTAINER_NAME'),
+            'thumbnail_container': os.environ.get('AZURE_THUMBNAIL_CONTAINER_NAME')
+        }
+    
+    @staticmethod
+    def generate_sas_url(account_name, container_name, blob_name, account_key, permission, expiry_hours):
```

Ruff (0.8.2)

Missing return type annotation for staticmethod `generate_sas_url`

Add return type annotation: `str`

(ANN205)